generators/e2e/templates/cypress.json.ejs

```diff
@@ -0,0 +1,10 @@
+{
+  "baseUrl": "http://localhost:8080",
```

Comment:
The port should be from the port constant

Reply:
Done :)